specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2022-02-14/imagebuilder.json

```diff
@@ -1376,7 +1376,7 @@
       "description": "Represents an output that was created by running an image template.",
       "allOf": [
         {
-          "$ref": "#../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
```

Comment:
If "#" was present in the ADO version then how was auto-rest still able to find the ProxyResource and generate correct model for us. Removing "#" is not an incorrect thing to do but I am just curios how our model worked with "#" in the path.

Reply:
the spec file we use to generate datamodels in ADO actually uses a different path than what is seen above since it's stored in a different directory (see swagger/api/2022-02-14/imagebuilder.server.json).

Reply:
https://swagger.io/docs/specification/using-ref/
```
Local Reference – $ref: '#/definitions/myElement' # means go to the root of the current document and then find elements definitions and myElement one after one.
```